docs/milestones.md

```diff
@@ -1,13 +1,13 @@
 # Milestones
 
-[[M0] Fichero donde se muestran indicadores que aunan datos para cada equipo](https://github.com/LuisMart7/ProBusiness/milestone/1)
-  
-Se definirá un fichero que sea visible a los clientes donde se muestren distintos ratios y porcentajes basados en modelos estadísticos que parten de los distintos datos que existen de cada equipo participante de la quiniela y quinigol (equipos de Liga Santander y Liga Smartbank). A partir de estos resultados, pondremos a funcionar el algoritmo de recomendación de combinación de la quiniela y quinigol.
+[[M0] Implementación del modelo de datos para cada equipo](https://github.com/LuisMart7/ProBusiness/milestone/1)
+
+Diseño de un modelo de datos basado en una estructura que nos permita organizar la información relevante de cada equipo atendiendo a distintos criterios como listado de partidos anteriores con información de los mismos, estado actual de la plantilla, competiciones en juego, etc.
   
 [[M1] Algoritmo de sugerencia de la probabilidad del resultado](https://github.com/LuisMart7/ProBusiness/milestone/2)
   
-Este algoritmo nos permitirá, a partir de los datos que tenemos, definir el porcentaje de probabilidad de resultados; y, por lo tanto, recomendar el mejor resultado posible para cada partido, útil para conocer la mejor combinación para la quiniela; así como las distintas opciones según el presupuesto provisto por el usuario.
+Este algoritmo nos permitirá, a partir del modelo de datos que hemos implementado en M0, definir el porcentaje de probabilidad de resultados entre los distintos enfrentamientos; y, por lo tanto, recomendar el mejor resultado posible para cada partido, útil para conocer la mejor combinación para la quiniela; así como las distintas opciones según el presupuesto provisto por el usuario, es decir; número de columnas, número de dobles o triples y elige 8. 
   
 [[M2] Algoritmo que permita conocer el resultado exacto de cada partido](https://github.com/LuisMart7/ProBusiness/milestone/3)
```

Comment:
No sé yo si esto se podrá hacer. Por favor, no pongáis cosas que no vayáis a hacer en la asignatura.